apps/verticalhome/js/stores/item.js

```diff
@@ -194,9 +195,13 @@
 
           objectStore.createIndex('index', 'index', { unique: true });
           isEmpty = true;
+
           var objectSV = db.createObjectStore(DB_SV_APP_STORE_NAME,
             { keyPath: 'manifestURL' });
           objectSV.createIndex('indexSV', 'indexSV', { unique: true });
+
+          var pinAppStore = db.createObjectStore(DB_PIN_APP_STORE, { keyPath: 'index'});
+          pinAppStore.createIndex('index', 'index', { unique: true });
```

Comment:
Maybe I'm misremembering my indexedDB, but I didn't think you had to create an index for the primary index of an object store....  I see the existing code does that, but I thought that this keyPath thing would mean you didn't have to create the index.  Though I would have thought that your keypath would be the manifest URL or something...

Maybe I'm just not clear on what you'r doing here.